nautobot/core/management/commands/populate_database.py

```diff
@@ -0,0 +1,69 @@
+from django.core.management import call_command
```

Comment:
IMO this should be named `populate_test_database` just so it's crystal clear that it is for testing.

Reply:
I'm 50/50 on this. That might also mislead people into thinking that it always populates a test database, rather than populating whatever database is currently live?

Reply:
Maybe something like "generate_test_data"?